lib/datasource/remote/api/key_accounts_repository.dart

```diff
@@ -3,18 +3,22 @@ import 'package:http/http.dart' as http;
 import 'package:seeds/datasource/remote/api/http_repo/http_repository.dart';
 
 class KeyAccountsRepository extends HttpRepository {
-  Future<Result<dynamic>> getKeyAccounts(String publicKey) {
```

Comment:
Same we need to use the old method to find accounts by key.... (see above)

Reply:
(see above)